stl/inc/ostream

```diff
@@ -454,6 +454,13 @@ public:
         return *this;
     }
 
+#if _HAS_CXX23 // P1147R1 "Printing volatile Pointers"
+    template <class = void> // TRANSITION, ABI
+    basic_ostream& operator<<(const volatile void* _Val) {
+        return *this << (const_cast<const void*>(_Val));
+    }
+#endif // _HAS_CXX23
+
 #if _HAS_CXX17 // LWG-2221 "No formatted output operator for nullptr"

```

Comment:
(Pre-existing) If someone wants to know how this function was added to the Standard they can look it up elsewhere:
```suggestion
#if _HAS_CXX17
```